src/OrbitVulkanLayer/SubmissionTrackerTest.cpp

```diff
@@ -1388,7 +1390,7 @@ TEST_F(SubmissionTrackerTest, CanLimitNestedDebugMarkerDepthPerCommandBuffer) {
 
   Color expected_color{1.f, 0.8f, 0.6f, 0.4f};
 
-  producer_->StartCapture();
+  producer_->StartCapture(1);
```

Comment:
Nit: Maybe define a constant kMaxDepth = 1 to indicate the meaning of "1" here.

Reply:
Done.